modules/GenebassGeneBurden.py

```diff
@@ -50,13 +51,13 @@ def main(genebass_data: str, spark_instance: SparkSession) -> DataFrame:
     # Write output
     evd_df = parse_genebass_evidence(genebass_df)
 
-    if evd_df.filter(col('resourceScore') == 0).count() != 0:
-        logging.error('There are evidence with a P value of 0.')
+    if evd_df.filter(F.col('resourceScore') == 0).count() != 0:
+        logging.assertion('There are evidence with a P value of 0.')
```

Comment:
I'm not sure if `logging.assertion` is a thing.. have you tried this?

Reply:
Thank you for flagging this. I got it confused with `logging.exception`